src/components/Button/Button.css

```diff
@@ -11,7 +11,14 @@
         text-xs
         text-tealA11y
         uppercase
-        leading-tight;
+        leading-tight
+        align-middle;
+
+  .Icon {
+    height: 9px;
```

Comment:
where does `9px` come from? seems like a magic number, could we try either `8` or `12` this will help keep it consistent with the spacing system